.github/workflows/vcpkg_ci.yml

```diff
@@ -127,4 +127,105 @@ jobs:
         uses: actions/upload-artifact@v1
         with:
           name: macos-11.0_llvm${{ matrix.llvm }}_tgz_package
-          path: ${{ steps.package_names.outputs.TGZ_PACKAGE_PATH }}
\ No newline at end of file
+          path: ${{ steps.package_names.outputs.TGZ_PACKAGE_PATH }}
+
+
+
+
+  release_linux:
+    # Do not run the release procedure if any of the builds has failed
+    needs: [ build_linux, build_mac ]
+    runs-on: ubuntu-20.04
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+
+    steps:
+      - name: Download all artifacts
+        uses: actions/download-artifact@v2
+
+      - name: Draft the new release
+        id: create_release
+        uses: actions/create-release@v1
+
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Version ${{ github.ref }}
+          draft: true
+          prerelease: true
```

Comment:
Do these need to change?

Reply:
This was done on purpose, so that we still have a chance to edit the release before it goes public